src/Microsoft.DocAsCode.Build.Engine/SingleDocumentBuilder.cs

```diff
@@ -75,12 +76,16 @@ private Manifest BuildCore(DocumentBuildParameters parameters)
                     parameters.Files.DefaultBaseDir);
                 if (ShouldTraceIncrementalInfo)
                 {
+                    string configHash = ComputeConfigHash(parameters);
+                    string templateHash = parameters.TemplateManager.GetTemplatesHash();
```

Comment:
>TemplateManager [](start = 53, length = 15)

As this is part of parameters, does it already included in configHash?
And each version shares the same TemplateManager, can it be calculated in DocumentBuilder?